filebeat/docs/fields.asciidoc

```diff
@@ -64,270 +64,183 @@ Contains fields for the Apache2 HTTPD access logs.
 --
 type: keyword
 
-Client IP address.
+Client IP address or hostname.
```

Comment:
Just curiosity, can the client host be logged as a hostname by apache?

Reply:
Yes, the position of  `%h` in the logs can be a hostname if the option `HostnameLookups` is set to `On`

Reply:
(default is Off)

Reply:
Oh interesting :+1: